nrf52810-hal/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "nrf52810-hal"
+version = "0.1.0"
+edition = "2018"
+description = "HAL for nRF52810 microcontrollers"
+repository = "https://github.com/nrf-rs/nrf52-hal"
+authors = [
+    "James Munns <james@onevariable.com>",
+    "Hanno Braun <hanno@braun-robotics.com>",
+    "John Scarrott <johnps@outlook.com>",
+    "Wez Furlong <wez@wezfurlong.org>",
+]
+categories = ["embedded", "hardware-support", "no-std"]
+keywords = ["arm", "cortex-m", "nrf52", "hal", "nrf52810"]
+license = "MIT OR Apache-2.0"
+
+[dependencies]
+cortex-m = "0.5.8"
+nb = "0.1.1"
+nrf52810-pac = "0.1.1"
```

Comment:
Where does this crate live? Should we move this into the nrf-rs organization?

Reply:
Whatever you want, it would be nice to have everything under the 1 organisation